lib/registrant.rb

```diff
@@ -0,0 +1,21 @@
+class Registrant
+attr_reader :name, :age, :permit, :license_data
+  def initialize(name, age, permit = false)
+    @name = name
+    @age = age
+    @permit = permit
+    @license_data = {
+      :written=>false,
+      :license=>false,
+      :renewed=>false
+    }
+  end
+
+  def permit?
+    @permit == true 
```

Comment:
Since @permit holds either true or false, we don't need to do the comparison with true. We could just have `@permit` on line 15.